src/index.html

```diff
@@ -11,7 +11,198 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      class="lines"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="form"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="Surename">Surname:
+            <input
+              type="text"
+              id="Surename"
+              name="Surname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="Name">
+              Name:
+            <input type="text"
+              id="Name"
+              name="Name"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div  class="form-field">
+          <label for="old">How old are You?
+
+            <input type="number"
+              name="howold"
+              id="old"
+              value="12"
+              min="1"
+              max="100"
+            >
```

Comment:
Please, apply such code styling for other html elements
```suggestion
            <input 
              type="number"
              name="howold"
              id="old"
              value="12"
              min="1"
              max="100"
            >
```